pkg/clusters/addons/kong/vars.go

```diff
@@ -0,0 +1,45 @@
+package kong
+
+// -----------------------------------------------------------------------------
+// Kong Addon - Defaults
+// -----------------------------------------------------------------------------
+
+const (
+	// KongHelmRepoURL indicates the URL that will be used for pulling Kong Helm charts.
+	KongHelmRepoURL = "https://charts.konghq.com"
+
+	// DefaultDBMode indicates which dbmode for the Kong proxy should be used if no other is specified.
+	DefaultDBMode = "none"
```

Comment:
Should this be `dbless` to match the value used in `pkg/clusters/addons/kong/dbmode.go`?

Doesn't look like we use this constant yet but the discrepancy seems off.